### PR TITLE
xbutil/xbmgmt mapping update

### PR DESCRIPTION
**Pull request by uday610**
Updates: 
  1) xbutil version     --> xbutil --version
  2) xbmgmt version --> xbmgmt --version
  3) xbutil --p2p ...    --> xbutil configure --p2p ...
  4) xbutil --host-mem --> xbutil configure --host-mem ...

### CODE REVIEW

File: src/runtime_src/doc/toc/xbtools_map.rst [hunk 1]
```diff
@@ -18,152 +18,176 @@ You may refer the documentation of the new tools in the following links :ref:`xb
...
+|            |             |                   |                      |
+|            |             |                   |``--output <file>``   |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|``dump``    |``examine``  |                   |``--format <type>``   |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|``top``     |``top``	   |``-d <bdf>``       |``--device <bdf>``    |
```
Comment (AShivangi) [inline src/runtime_src/doc/toc/xbtools_map.rst:78]: We need to remove top as it will not work without the "--legacy" switch

**Review by AShivangi — COMMENTED** [review]
Would you like to update the xbutil page with the `configure` subcommand?

File: src/runtime_src/doc/toc/xbtools_map.rst (hunk 1 above)
Comment (uday610) [reply src/runtime_src/doc/toc/xbtools_map.rst]: Removed top

**Comment by uday610** [comment]
> Would you like to update the xbutil page with the `configure` subcommand?

I will send separate PRs for the main pages later. This is for the mapping page

**Review by AShivangi — APPROVED** [review]

**Pull request merged**